theblues/charmstore.py

```diff
@@ -20,7 +20,7 @@
 class CharmStore(object):
     """A connection to the charmstore."""
 
-    def __init__(self, url, macaroons=None, timeout=None, verify=True):
+    def __init__(self, url, macaroons=None, timeout=3.05, verify=True):
```

Comment:
Would be nice if the magic number of 3.0.5 were defined in one place rather than having a constant everywhere.